editor/src/components/canvas/controls/reorder-slider-control.tsx

```diff
@@ -127,7 +127,7 @@ export const ReorderSliderControl = controlForStrategyMemoized(
                 background: colorTheme.bg0.value,
                 boxShadow: `inset 0px 0px 0px ${0.5 / scale}px ${colorTheme.border3.value} , 0px ${
                   2 / scale
-                }px ${4 / scale}px 0px ${colorTheme.fg6.o(50).value}`,
+                }px ${4 / scale}px 0px ${colorTheme.fg650.value}`,
```

Comment:
I think this naming convention is going to cause confusion. I know it's pretty verbose, but maybe it would be better to go with `fg6opacity50` or `fg6o50`? I don't think it's important though, and these keys are in a specific part of the theme that will (hopefully) go soon, so I'm easy either way tbh.

Reply:
Anything but 650, please :D Probably more verbose is better since we've got autocomplete

Reply:
went verbose